### PR TITLE
EDU-74 Update readme

### PR DESCRIPTION
**Pull request by laurentiu-ilici**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,44 +4,51 @@
...
     username: 'test',
     password: 'test',
     email: 'test@test.com'
-  }
+  },
+  exposeErrorDetails: true //set to false in order to hide error details such as stack trace on productive systems
 });
 ~~~
 
+This should run both the entire application and provide you with an admin user as defined on the initialUser setting. In order to create other admins please consult the users page in the application. Here, admins can promote other users to the admin role.
```
Comment (ahelmberger) [inline README.md:50]: I don't think this is currently possible

Comment (laurentiu-ilici) [reply README.md:50]: Creating other admins?

Comment (ahelmberger) [reply README.md]: Yep, not possible on the UI currently

Comment (laurentiu-ilici) [reply README.md]: Wow, yes, you are right... I deleted that part.

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**